### PR TITLE
Upgrade to official Jaeger chart

### PR DESCRIPTION
**Pull request by chrissng**

### CODE REVIEW

File: modules/jaeger/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Jaeger tracing
+
+This TF module deploys a Jaeger backend that includes an Elasticsearch cluster. Also, it schedules a cron job that cleans up tracing data that is older than 4 weeks, i.e. it has a default retention period of 4 weeks.
+
+## Chart repository
+
+Helm chart that is used resides in Jaeger OSS project's official repository. To add the repo, run:
+```bash
+helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
```
Comment (lawliet89) [inline modules/jaeger/README.md]: Use the [`helm_repository`](https://www.terraform.io/docs/providers/helm/repository.html) data source.

File: modules/jaeger/variables.tf [hunk 2]
```diff
@@ -47,100 +47,183 @@ variable "jaeger_ui_ingress_annotations" {
...
     limits = {
-      cpu    = "1"
-      memory = "1536Mi"
+      cpu    = "200m"
+      memory = "128Mi"
     }
     requests = {
       cpu    = "25m"
-      memory = "512Mi"
+      memory = "32Mi"
```
Comment (lawliet89) [inline modules/jaeger/variables.tf]: Request = limit

```suggestion
      memory = "128Mi"
```

File: modules/jaeger/variables.tf [hunk 3]
```diff
@@ -47,100 +47,183 @@ variable "jaeger_ui_ingress_annotations" {
...
-      cpu    = "1"
-      memory = "1536Mi"
+      cpu    = "512m"
+      memory = "256Mi"
     }
     requests = {
-      cpu    = "25m"
-      memory = "512Mi"
+      cpu    = "50m"
+      memory = "64Mi"
```
Comment (lawliet89) [inline modules/jaeger/variables.tf]: Request = limit

```suggestion
      memory = "256Mi"
```

File: modules/jaeger/variables.tf [hunk 4]
```diff
@@ -47,100 +47,183 @@ variable "jaeger_ui_ingress_annotations" {
...
     limits = {
-      cpu    = "1"
-      memory = "2560Mi"
+      cpu    = "100m"
+      memory = "128Mi"
     }
     requests = {
       cpu    = "25m"
-      memory = "1536Mi"
+      memory = "16Mi"
```
Comment (lawliet89) [inline modules/jaeger/variables.tf]: Request = limit

```suggestion
      memory = "128Mi"
```

File: modules/jaeger/variables.tf [hunk 5]
```diff
@@ -47,100 +47,183 @@ variable "jaeger_ui_ingress_annotations" {
...
     limits = {
-      cpu    = "200m"
-      memory = "128Mi"
+      cpu    = "1"
+      memory = "1536Mi"
     }
     requests = {
       cpu    = "25m"
-      memory = "32Mi"
+      memory = "512Mi"
```
Comment (lawliet89) [inline modules/jaeger/variables.tf]: Request = limit

```suggestion
      memory = "1536Mi"
```

File: modules/jaeger/variables.tf [hunk 6]
```diff
@@ -47,100 +47,183 @@ variable "jaeger_ui_ingress_annotations" {
...
-      cpu    = "512m"
-      memory = "256Mi"
+      cpu    = "1"
+      memory = "1536Mi"
     }
     requests = {
-      cpu    = "50m"
-      memory = "64Mi"
+      cpu    = "25m"
+      memory = "512Mi"
```
Comment (lawliet89) [inline modules/jaeger/variables.tf]: Request = limit


```suggestion
      memory = "1536Mi"
```

File: modules/jaeger/variables.tf [hunk 7]
```diff
@@ -47,100 +47,183 @@ variable "jaeger_ui_ingress_annotations" {
...
     limits = {
-      cpu    = "100m"
-      memory = "128Mi"
+      cpu    = "1"
+      memory = "2560Mi"
     }
     requests = {
       cpu    = "25m"
-      memory = "16Mi"
+      memory = "1536Mi"
```
Comment (lawliet89) [inline modules/jaeger/variables.tf]: Request = limit


```suggestion
      memory = "2560Mi"
```

**Review by lawliet89 — CHANGES_REQUESTED** [review]
Mostly looks OK.

File: modules/jaeger/README.md (hunk 1 above)
Comment (chrissng) [reply modules/jaeger/README.md]: Opps, why didnt i think about that. will do

Comment (chrissng) [reply modules/jaeger/README.md]: Done

**Review by lawliet89 — APPROVED** [review]

**Pull request merged**